internal/agent/service/service_windows.go

```diff
@@ -100,3 +100,31 @@ func (d *daemon) run() {
 		}
 	}
 }
+
+// runAgentCmd will run the agent process and wait it to exit. The process will be added
+// to an Windows job object to handle the child processes.
+func runAgentCmd(cmd *exec.Cmd) error {
+	jobObject, err := NewJob()
+	if err != nil {
+		log.Warnf("Failed to create Job Object for Agent: %v", err)
```

Comment:
log entries should start in lower case

Reply:
I think @theletterf changed them all to start with an Upper case to be consistent.